bigbluebutton-html5/imports/ui/components/breakout-room/breakout-room/styles.ts

```diff
@@ -345,6 +341,35 @@ const ErrorMessage = styled.div`
   max-height: .93rem;
 `;
 
+const NotAssignedWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+  background-color: ${colorBlueLightest};
+  border-radius: ${borderRadius};
+  padding: 1rem 1.5rem;
+  flex-grow: 1;
+
+  & > i {
+    font-size: 3rem;
+    color: ${colorPrimary};
+  }
+`;
+
+const NotAssignedMessage = styled.p`
+  font-size: ${fontSizeBase};
+  color: ${colorText};
+  margin: 0;
```

Comment:
Missing text-align: center;

```suggestion
  margin: 0;
  text-align: center;
```